app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/SkillDetailsFragment.kt

```diff
@@ -118,6 +119,40 @@ class SkillDetailsFragment : Fragment(), ISkillDetailsView {
         }
     }
 
+    private fun setReportButton() {
+        //Only enable the feature when the user is logged in
+        if (PrefManager.getToken() != null) {
+            reportSkill.visibility = View.VISIBLE
+            reportSkill.setOnClickListener {
+                val dialogBuilder = AlertDialog.Builder(requireContext())
+                val view = layoutInflater.inflate(R.layout.alert_report_skill, null)
+                val reportMessage = view.findViewById(R.id.reportMessage) as EditText
```

Comment:
Please check if ```reportMessage``` can be referenced directly in the Kotlin file.

Reply:
@arundhati24 we have to inflate the xml and then reference from that view, that's why we have to apply findViewById.